autovacuum_message_attachment/models/base.py

```diff
@@ -0,0 +1,13 @@
+# Copyright (C) 2019 Akretion
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+from odoo import fields, models
+
+
+class Base(models.AbstractModel):
+    _inherit = "base"
+
+    attachment_ids = fields.One2many(
+        'ir.attachment', 'res_id', string='Attachments',
+        domain=lambda self: [('model', '=', self._name)], auto_join=True
```

Comment:
It's res_model, not model, that's the main reason for travis failing.

Reply:
Well strange the test `test_attachment_vacuum_rule` did not fail...

Reply:
We probably should add test for rule with no linked model..